src/test/resources/cliointegrationtest/clio_test.py

```diff
@@ -11,6 +11,12 @@
 clio_http_port = os.environ.get('CLIO_HTTP_PORT', 8080)
 clio_http_uri = '{0}://{1}:{2}'.format(clio_http_scheme, clio_http_host, clio_http_port)
 
+elasticsearch_http_scheme = os.environ.get('ELASTICSEARCH_HTTP_SCHEME', 'http')
+elasticsearch_http_host = os.environ.get('ELASTICSEARCH_HTTP_HOST', 'localhost')
+elasticsearch_http_port = os.environ.get('ELASTICSEARCH_HTTP_PORT', 9200)
+elasticsearch_http_uri = '{0}://{1}:{2}'.format(
```

Comment:
Maybe make a method or string constant to avoid the duplication when creating a URI from scheme/host/port?